packit_service/worker/jobs.py

```diff
@@ -81,23 +132,36 @@ def get_handlers_for_event(
         ):
             jobs_matching_trigger.append(job)
 
-    handlers_triggered_by_comment = None
-    handlers_triggered_by_check_rerun = None
+    return jobs_matching_trigger
 
-    if isinstance(event, AbstractCommentEvent):
-        handlers_triggered_by_comment = get_handlers_for_comment(
-            event.comment, packit_comment_command_prefix
-        )
 
-        if handlers_triggered_by_comment and not isinstance(
-            event, PullRequestCommentPagureEvent
-        ):
-            event.comment_object.add_reaction(COMMENT_REACTION)
+def get_handlers_for_event(
+    event: Event, package_config: PackageConfig, packit_comment_command_prefix: str
+) -> Set[Type[JobHandler]]:
+    """
+    Get all handlers that we need to run for the given event.
 
-    if isinstance(event, CheckRerunEvent):
-        handlers_triggered_by_check_rerun = get_handlers_for_check_rerun(
-            event.check_name_job
-        )
+    We need to return all handler classes that:
+    - can react to the given event AND
+    - are configured in the package_config (either directly or as a required job)
```

Comment:
I thought about this for some time, but what about returning pair of handler and job config so we don't need to do the similar search again in `get_config_for_handler_kls`.

(Yes, it does not need to be solved here...;)

Reply:
yes we can do that here but I would keep that function since we use it in `babysit.py` and it seems like it's doing its job there. Should I rather move the method to `babysit.py` then since it should be used only there or keep it in `jobs.py`? WDYT?

Reply:
Ok, then it makes sense to preserve it... And you can leave it in the current space I think since it's related to the function you've refactored.